build.gradle

```diff
@@ -742,7 +742,8 @@ subprojects {
                                 "-workDir:${jtregOutput}/${name}/work",
                                 "-reportDir:${jtregOutput}/${name}/report",
                                 "-verbose:error,fail",
-                                "-javacoptions:-g",
+                                // Don't add debugging information
```

Comment:
@smillst Is this change desirable?  Does it belong as part of this pull request?

Reply:
If you revert this change, then the jtreg tests added by this pull request will fail.  So if we don't make this change here, then the test will have to be rewritten.

It took me a while to figure out that this was necessary to get the test case I added to fail.  (I could reproduce the problem locally, but not in a test case.)   In the future, we may have other test case the behave differently if `-g` is passed, so it may be worthwhile not to pass it.  But, I also don't know what the benefit of passing `-g` to the jtreg test was in the first place.